qc/workflow_exome.cwl

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Exome QC workflow"
+requirements:
+    - class: SubworkflowFeatureRequirement
+inputs:
+    bam:
+        type: File
+        secondaryFiles: [^.bai]
```

Comment:
Yeah, it's weird. When I look at an actual output directory of this workflow the file is named Final.bai. Not sure what is happening there.